pages.pl/osx/say.md

```diff
@@ -0,0 +1,28 @@
+# say
+
+> Czyta na głos.
+> Więcej informacji: <https://ss64.com/osx/say.html>.
+
+- Powiedz na głos:
+
+`say "{{Lubię jeździć na rowerze.}}"`
+
+- Przeczytaj z pliku:
+
+`say --input-file={{plik.txt}}`
+
+- Przeczytaj używając konkretnego głosu i prędkości mowy:
+
+`say --voice={{głos}} --rate={{słowa_na_minutę}} "{{Przepraszam Dave, ale nie mogę Ci na to pozwolić.}}"`
+
+- Pokaż listę dostępnych głosów:
+
+`say --voice="?"`
+
+- Powiedz coś Angielskim głosem:
+
+`say --voice=Alex "{{Here's to the Crazy Ones.}}"`
```

Comment:
The following code fix could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/say.md) is updated:

```suggestion
`say --voice={{Alex}} "{{Here's to the Crazy Ones.}}"`
```

`Alex` can be converted to a placeholder as `--voice` allows several values to choose.

Reply:
Changed to placeholder, also changed wording a bit and mentioned in the previous command (`--voice=?`) that different voices cause different spoken language. I'll add a PR for this change in english page.